secure_fw/spm/cmsis_func/spm_func.c

```diff
@@ -80,6 +76,7 @@ void tfm_spm_seal_psp_stacks(void)
      *                                      |                         |
      * Image$$TFM_SECURE_STACK$$ZI$$Base->  +-------------------------+
      */
+    uint32_t *tfm_secure_stack_seal = ((uint32_t *)&REGION_NAME(Image$$, TFM_SECURE_STACK, $$ZI$$Limit)[-1]) - 2;
```

Comment:
It's kind of funny that the ASCII art above has a more clearer code than the actual code.